site/content/en/docs/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'CVAT Documentation'
+linkTitle: 'Documentation'
+no_list: true
+menu:
+  main:
+    weight: 20
+---
+
+CVAT is free, online, interactive video and image annotation tool for computer vision.
+It is used by our team to annotate millions of objects with different properties.
+Many UI and UX decisions are based on feedbacks from professional data annotation team.
+Try it online [cvat.org](https://cvat.org).
+
+<section id="docs">
+
+{{< blocks/section color="docs" >}}
+
+{{% blocks/feature icon="fa-server" title="[Installation Guide](../docs/for-users/installation/)" %}}
+
+CVAT installation guide for different operating systems.
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fa-book" title="[User's Guide](../docs/for-users/user-guide/)" %}}
+
+This multipage document contains information on how to work with the CVAT user interface.
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fa-question" title="[FAQ](../docs/for-users/faq/)" %}}
+
+Answers to frequently asked questions.
+
+{{% /blocks/feature %}}
+
+<!--lint disable maximum-line-length-->
+
+{{% blocks/feature icon="fa-magic" title="[Installation Auto Annotation](../docs/for-users/installation_automatic_annotation/)" %}}
+
+This page provides information about the installation of components needed for semi-automatic and automatic annotation.
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fa-terminal" title="[For Developers](../docs/for-developers/)" %}}
+
+This section contains documents for CVAT developers.
```

Comment:
I am not sure that topics under this link are for developers.
It seems to me that they are for system administrators, AI researchers and any other advanced users.